components/NotesTable.tsx

```diff
@@ -29,7 +29,7 @@ export default function NotesTable({ notes, query }: Props) {
         </div>
       )}
       {!!notes?.length ? (
-        <div className="grid md:grid-cols-2 lg:grid-cols-3 gap-5 lg:gap-10">
+        <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-5 lg:gap-10">
```

Comment:
Isnt this what grid defaults to?

Reply:
Was the only way to whip Safari into place, without it the text didn't truncate 🙃